### PR TITLE
Add Build URL as an ENV var

### PR DESCRIPTION
**Pull request by BaseInfinity**
This will allow the build URL to be available as an ENV var

### CODE REVIEW

File: test/shell.test.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ describe("Shell plugin", function() {
             'BUILD_REPO',
             'BUILD_COMMIT',
             'BUILD_BRANCH',
-            'BUILD_STATUS'],
+            'BUILD_STATUS',
+            'BUILD_URL'],
```
Comment (danielbeardsley) [inline test/shell.test.js]: You need to also add to the list of *expected* values on line 55.

**Comment by danielbeardsley** [comment]
CR 👍

**Comment by danielbeardsley** [comment]
CR 👍

**Comment by BaseInfinity** [comment]
ty for doing that =)

**Pull request merged**